src/backend/distributed/citus.sql

```diff
@@ -103,6 +103,27 @@ SET search_path = 'pg_catalog';
 
 /* master_* functions */
 
+CREATE FUNCTION citus_table_size(relation_name text)
+    RETURNS integer
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$citus_table_size$$;
+COMMENT ON FUNCTION citus_table_size(relation_name text)
+    IS 'get the size of distributed tables';    
```

Comment:
All function comments are "get the size of distributed tables". We should explain what each function does differently from the other.